beetmoverscript/templates/fennec_nightly.yml

```diff
@@ -128,5 +128,9 @@ mapping:
       destinations:
         - latest-{{ branch }}-{{ platform }}/{{ locale_prefix }}robocop.apk
         - {{ upload_date }}-{{ branch }}-{{ platform }}/{{ locale_prefix }}robocop.apk
-
+    buildhub.json:
+      s3_key: fennec-{{ version }}.{{ locale }}.{{ filename_platform }}.buildhub.json
+      destinations:
+        - latest-{{ branch }}-{{ platform }}/{{ locale_prefix }}.buildhub.json
```

Comment:
Let's stick to `fennec-{{ version }}.{{ locale }}.{{ filename_platform }}.buildhub.json` to for these filenames. AFAIK the `{{ locale_prefix }}` evaluates to empty string for `en-US` so that file will be slightly confusing, at least in the `latest...` dirs. I'd rather opt for 

```
latest-{{ branch }}-{{ platform }}/fennec-{{ version }}.{{ locale }}.{{ filename_platform }}.buildhub.json
{{ upload_date }}-{{ branch }}-{{ platform }}/fennec-{{ version }}.{{ locale }}.{{ filename_platform }}.buildhub.json
```